timemachine/potentials/jax_utils.py

```diff
@@ -210,7 +210,7 @@ def bernoulli_logpdf(log_p_i, z_i) -> float:
     b = jnp.array([jnp.ones(n), -jnp.ones(n)])
     log_1_minus_p_i = logsumexp(a=a, b=b, axis=0)
 
-    return jnp.sum(jnp.where(z_i, log_p_i, log_1_minus_p_i))
+    return cast(float, jnp.sum(jnp.where(z_i, log_p_i, log_1_minus_p_i)))
```

Comment:
This was causing a mypy error, since the return type of `jnp.sum` is `jax.Array | float`. I was thinking it was a result of the jax upgrade, but am not sure now. Will double check.

Reply:
As discussed offline, it looks like this was not caught before because the pre-commit mypy check did not declare JAX as a dependency. Planning to leave this fix in for now and address the mypy issue in a later PR.